src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppElasticBeanstalkWindows.recipe

```diff
@@ -0,0 +1,813 @@
+                    "Type": "Bool",
+                    "DefaultValue": true,
+                    "AdvancedSetting": false,
+                    "Updatable": false
+                },
+                {
+                    "Id": "ApplicationName",
+                    "Name": "Application Name",
+                    "Description": "The Elastic Beanstalk application name.",
+                    "Type": "String",
+                    "DefaultValue": "{StackName}",
+                    "AdvancedSetting": false,
+                    "Updatable": false,
+                    "DependsOn": [
+                        {
+                            "Id": "BeanstalkApplication.CreateNew",
+                            "Value": true
+                        }
+                    ],
+                    "Validators": [
+                        {
+                            "ValidatorType": "Regex",
+                            "Configuration": {
+                                "Regex": "^[^/]{1,100}$",
+                                "AllowEmptyString": true,
+                                "ValidationFailedMessage": "Invalid Application Name. The Application name can contain up to 100 Unicode characters, not including forward slash (/)."
+                            }
+                        },
+                        {
+                            "ValidatorType": "ExistingResource",
+                            "Configuration": {
+                                "ResourceType": "AWS::ElasticBeanstalk::Application"
+                            }
+                        }
+                    ]
+                },
+                {
+                    "Id": "ExistingApplicationName",
+                    "Name": "Application Name",
+                    "Description": "The Elastic Beanstalk application name.",
+                    "Type": "String",
+                    "TypeHint": "ExistingBeanstalkApplication",
+                    "DefaultValue": "{StackName}",
+                    "AdvancedSetting": false,
+                    "Updatable": false,
+                    "DependsOn": [
+                        {
+                            "Id": "BeanstalkApplication.CreateNew",
+                            "Value": false
+                        }
+                    ],
+                    "Validators": [
+                        {
+                            "ValidatorType": "Regex",
+                            "Configuration": {
+                                "Regex": "^[^/]{1,100}$",
+                                "AllowEmptyString": true,
+                                "ValidationFailedMessage": "Invalid Application Name. The Application name can contain up to 100 Unicode characters, not including forward slash (/)."
+                            }
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "Id": "BeanstalkEnvironment",
```

Comment:
Since this is a new recipe, let's do it right and have 1 setting and not a nested structure

Reply:
Done